pkg/stockings/stock.go

```diff
@@ -1,8 +1,9 @@
 package stockings
 
 import (
-	"github.com/shopspring/decimal"
 	"time"
+
+	"github.com/shopspring/decimal"
```

Comment:
doesn't seem like a gofmt format either

Reply:
will run go fmt in git as well, maybe rishov should just go fmt it too

Reply:
we're gonna need to discord over this to see how your IDE is running go fmt